kbase-extension/scss/partials/_narrative.scss

```diff
@@ -1357,32 +1401,15 @@ img.remove {
   background-color: fbfaed;
 }
 
-div {
-  &.metadata {
-    display: block;
-    width: 200px;
-    padding: 0 10px;
-    height: 100px;
-    float: left;
-  }
-
-  &.social {
-    float: right;
-    width: 80px;
-    padding: 0 10px;
-    display: none;
-  }
-}
-
 p span.notification {
-  background-color: rgb(255 51 51);
-  color: rgb(255 255 255);
   padding: 4px 8px;
+  background-color: rgb(255 51 51);
   border-radius: 1em;
+  color: rgb(255 255 255);
 }
 
 /**
- * Bootstrap 3 submenu hack^H^H^H^H fix
+ * Bootstrap 3 submenu fix
```

Comment:
but how will we know that it's a l33t h@x anymore without this comment???

Reply:
You're right -- I'll restore the comment ASAP!